extension.ts

```diff
@@ -112,6 +112,15 @@ export async function activateInternal(ctx: vscode.ExtensionContext, perfStats:
     this.properties.isActivationEvent = 'true';
     this.measurements.mainFileLoad = (perfStats.loadEndTime - perfStats.loadStartTime) / 1000;
 
+    const extension = vscode.extensions.getExtension('PeterJausovec.vscode-docker');
+    if (extension) {
+      // temporary workaround for https://github.com/microsoft/vscode-docker/issues/1008
+      // We will essentially defer all functionality to the old publisher's extension if we detect both installed
+      vscode.commands.executeCommand('setContext', 'isOldPublisherInstalled', true);
```

Comment:
The context is global, right?  Key should be specific to the extension.

Reply:
Changed to `isOldDockerPublisherInstalled`